Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -13147,38 +13147,93 @@ def add_time_filter_to_query_parameter(query: str, last_fetch: datetime) -> str:
     return query + time_generated
 
 
-def add_unique_id_filter_to_query_parameter(query: str, last_id: str) -> str:
-    """append unique id filter parameter to original query parameter.
-    'seqno' is a 64-bit log entry identifier incremented sequentially; each log type has unique number space.
-    by adding this filter which gives us only entries that have larger id, we insure not have duplicates in our request.
+def find_largest_id_per_device(incident_entries: List[Dict[str, Any]]) -> Dict[str, str]:
+    """
+    This function finds the largest sequence id per device in the incident entries list.
+    Args:
+        incident_entries (List[Dict[str, Any]]): list of dictionaries representing raw incident entries
+    Returns:
+        new_largest_id: a dictionary of the largest sequence id per device
+    """
+
+    new_largest_id: Dict[str, str] = {}
```

Comment:
Can `new_largest_id` be of type `Dict[str, int]`?

Reply:
Since it was originally a string and all data kept in last_run is a string, I left it as is and converted to an int when necessary.